docs/book/src/user/quick-start.md

```diff
@@ -228,6 +228,11 @@ clusterctl init --infrastructure openstack
 
 Please visit the [Metal3 project][Metal3 getting started guide].
 
+{{#/tab }}
+{{#tab Packet}}
```

Comment:
Does Packet also need to be added to line 491 (CNI instructions) https://github.com/kubernetes-sigs/cluster-api/pull/3123/files#diff-dccb64095f5c5a09d32f119446301d12L486?